docs/supported-versions.asciidoc

```diff
@@ -1,11 +1,12 @@
-* link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[kubectl] 1.16-1.20
+* Kubernetes 1.16-1.20
 * OpenShift 3.11, 4.3-4.7
 * Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), and Amazon Elastic Kubernetes Service (EKS)
 * Elasticsearch, Kibana, APM Server: 6.8+, 7.1+
 * Enterprise Search: 7.7+
 * Beats: 7.0+
 * Elastic Agent: 7.10+
 * Elastic Maps Server: 7.11+
+* UBI images are only available from 7.10+
```

Comment:
Should we make it explicit that this applies to the certified Red Hat OpenShift Operator version of ECK? I am not sure how many people will make the logical leap from "UBI" to "this applies to the OpenShift certified operator"?